src/styles/variables.scss

```diff
@@ -0,0 +1,13 @@
+$header-height: 60px;
+$product-image-height: 134px;
+
+$hover-color: #00acdc;
+$nav-link-color: #060b35;
+$buy-button-color: rgb(0, 172, 220);
```

Comment:
The `$buy-button-color` and `$hover-color` seem to represent the same color but are defined using different formats (hex and rgb). It's better to be consistent with color formats. Consider using only one format for colors, preferably hex for simplicity and consistency.